.gitignore

```diff
@@ -1,29 +1,4 @@
-# Compiled Object files, Static and Dynamic libs (Shared Objects)
```

Comment:
Just .vscode line

Kirill.

-----Original Message-----
From: Darren Shepherd [notifications@github.com]
Received: Sunday, 24 Apr 2016, 19:11
To: rancher/sparse-tools [sparse-tools@noreply.github.com]
CC: Kirill Pavlov [kirill@rancher.com]; Comment [comment@noreply.github.com]
Subject: Re: [rancher/sparse-tools] Add build scripts (#11)


In .gitignore<https://github.com/rancher/sparse-tools/pull/11#discussion_r60857741>:

> @@ -1,29 +1,4 @@
> -# Compiled Object files, Static and Dynamic libs (Shared Objects)


Do you want me to keep the original one?

-
You are receiving this because you commented.
Reply to this email directly or view it on GitHub<https://github.com/rancher/sparse-tools/pull/11/files/55e25d674cb3d5723689d16ed00d9bf21b2bef71..8ddb09499be00ffc93dc637c3f121d8d73759104#r60857741>